starter-code/src/functions-and-arrays.js

```diff
@@ -1,20 +1,125 @@
 // Iteration #1: Find the maximum
 
-// Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function maxOfTwoNumbers(a,b){
+  if (a > b) {return a}
+  else if (a < b) {return b}
+  else {return a && b};
+}
 
-// Iteration #3: Calculate the sum
+
+
+
+
+
+
+/////////////////////////////////////////////////////////
+///////////////////// Iteration #2: Find longest word
+const words = [
+  'mystery', 
+  'brother', 
+  'aviator', 
+  'crocodile', 
+  'pearl', 
+  'orchard', 
+  'crackpot'];
+
+function findLongestWord(arg1) {
```

Comment:
esta funcion queda de la siguiente manera
`function findLongestWord(arg1) {
  if (!arg1.length ) {return null} // hasta aqui todo bien
  else{let word = ''
  for(let i = 0;i < arg1.length ;i++){
    if(arg1[i].length > word.length ){
      word = arg1[i]
    }
  }
  return word
}`